README.md

```diff
@@ -1 +1,33 @@
-# IPLive
\ No newline at end of file
+# Cric-Live-Scoreboard
+
+> - It is made using ReactJs 
+> - API used => [cric-api](https://www.cricapi.com/)
+> - app deployed at [cric-live](https://cric-liv.web.app/)
```

Comment:
- Use Capital A for app.
- Change the link of the app to the link given in the about section.

Reply:
okay!